tests/Responses/Expected/error/not_found_response.json

```diff
@@ -0,0 +1,4 @@
+{
+    "code": 404,
+    "message": @string@
```

Comment:
Without knowing anything about it: Should be in quotes?

Reply:
Well, both notation is valid.

Original Sylius developers usually use quotes for strings and not use for integers (https://github.com/Sylius/Sylius/blob/master/tests/Responses/Expected/product/create_response.json#L2).

I prefer to use quotes everywhere.

In this particular case, file `error/not_found_response.json` was just copied from original Sylius without any changes.